docs/docs/themes/getting-started.md

```diff
@@ -1,35 +1,56 @@
 ---
-title: Getting Started
+title: Using a Gatsby theme
 ---
 
-Scaffolding out a project that uses themes requires you to use a starter that's implemented with theming.
+To get started using themes, we'll use a starter that installs the official gatsby blog theme.
+
+A **regular Gatsby starter** creates a new Gatsby site that is preconfigured for a particular use case. The resulting site effectively forks off the starter — after it’s created, the site maintains no connection to the starter.
```

Comment:
An example would help illustrate this. Can we recommend a regular Gatsby starter here, like https://github.com/gatsbyjs/gatsby-starter-blog?